.circleci/config.yml

```diff
@@ -104,6 +104,30 @@ commands:
           cd ~/repo/tests
           ./travis-run-tests.sh << parameters.flags >>
 
+  # Build and push PyNE website
+  website_build_push:
+    description: "build PyNE website and push it"
```

Comment:
should this specify that the default "push" is the "test" push?

Reply:
I have a slight preference to keep this description small, focused of the general behavior, but don't have a strong opinion about this... (as it is explain in the README file and the default value if specified 4 line below...)

But if anyone feel strongly about this, I can definitely change it!

Reply:
If I did `website_build_push` I might expect it to actually update the website and then be confused why it was not. So specifying that it's a test push could be helpful? I'm not tied one way or the other either, but could help eliminate confusion.

Reply:
does it look better ?

Reply:
yes I think so